models/webhook/webhook_system.go

```diff
@@ -46,6 +46,19 @@ func GetSystemWebhooks(ctx context.Context, isActive util.OptionalBool) ([]*Webh
 		Find(&webhooks)
 }
 
+// GetSystemOrDefaultWebhooks returns all admin webhooks.
+func GetSystemOrDefaultWebhooks(ctx context.Context, isActive util.OptionalBool) ([]*Webhook, error) {
+	webhooks := make([]*Webhook, 0, 5)
+	if isActive.IsNone() {
+		return webhooks, db.GetEngine(ctx).
+			Where("repo_id=? AND org_id=?", 0, 0).
+			Find(&webhooks)
+	}
+	return webhooks, db.GetEngine(ctx).
+		Where("repo_id=? AND org_id=? AND is_active = ?", 0, 0, isActive.IsTrue()).
+		Find(&webhooks)
+}
+
```

Comment:
There's improvements to be made here. For example, if this exists, do we really still need `GetSystemWebhooks()`? Would you ever use both?